colorbleed/plugins/maya/publish/collect_instances.py

```diff
@@ -113,7 +113,11 @@ def process(self, context):
             # user interface interested in visualising it.
             self.log.info("Found: \"%s\" " % instance.data["name"])
 
-        context[:] = sorted(context)
+        # Sort/grouped by family (preserving local index)
+        grouped = sorted(enumerate(context), key=self.sorter)
+        context[:] = [x[1] for x in grouped]
+        # context[:] = sorted(context, key= lambda x: (x.data['families'],
```

Comment:
Remove the commented lines :)

Also, I personally wouldn't push the `sorted` method to the class because it's so specific to these specific lines of code. Which is related to just replacing a lambda in PEP08, see:

- https://www.python.org/dev/peps/pep-0008/#programming-recommendations
- https://stackoverflow.com/questions/25010167/e731-do-not-assign-a-lambda-expression-use-a-def

These can and are recommended even to be inline if they are just that, like a lambda.